examples/flutter_gallery/lib/demo/video_demo.dart

```diff
@@ -321,6 +329,7 @@ class _VideoDemoState extends State<VideoDemo>
 
   final GlobalKey<ScaffoldState> scaffoldKey = new GlobalKey<ScaffoldState>();
   final Completer<Null> connectedCompleter = new Completer<Null>();
+  bool unsupported = false;
```

Comment:
Could be inverted to avoid a negative boolean variable. `isSupported`?

Reply:
done